README.md

```diff
@@ -2,7 +2,7 @@
 
 BaseDAO is a generic smart contract on Tezos that enables a community to collectively govern resources, registries, or rules.
 
-The contract enables the creator to customize their DAO based on a number of attributes, and uses a ‘decision lambda’ to specify arbitrary code that can be governed by a DAO’s proposals.
+The contract enables the creator to customize their DAO based on a number of attributes, and uses a ‘decision callback’ to specify arbitrary code that can be governed by a DAO’s proposals.
 BaseDAO also includes Permit (TZIP-17) to enable off-chain voting.
 
 ## Documentation and configurations
```

Comment:
In this section there is a paragraph starting with:
```
BaseDAO uses a runtime configuration
```
which I believe is no longer correct

Reply:
Have change it.